### PR TITLE
feat(envoy): declare chunk post endpoint with retry policy

### PR DESCRIPTION
**Pull request by hlolli**

### CODE REVIEW

File: envoy/envoy.template.yaml [hunk 1]
```diff
@@ -69,6 +69,32 @@ static_resources:
...
                             prefix_rewrite: '/'
+                        - match:
+                            prefix: '/chunk'
+                            headers:
+                              - name: ':method'
+                                exact_match: 'POST'
+                          route:
+                            cluster: ario_gateways
+                            timeout: 30s
+                            retry_policy: &post_retry_policy
```
Comment (djwhitt) [inline envoy/envoy.template.yaml:80]: We don't need a retry policy here. The ar-io-node service handles that.

File: envoy/envoy.template.yaml [hunk 2]
```diff
@@ -69,6 +69,32 @@ static_resources:
...
+                              retriable_status_codes: 404
+                              retry_on: '5xx,reset,retriable-status-codes,connect-failure,refused-stream'
+                              num_retries: 3
+                              host_selection_retry_max_attempts: 3
+                              retry_host_predicate:
+                                name: envoy.retry_host_predicates.previous_hosts
+                                typed_config:
+                                  '@type': type.googleapis.com/envoy.extensions.retry.host.previous_hosts.v3.PreviousHostsPredicate
+                        - match:
+                            prefix: '/tx'
```
Comment (djwhitt) [inline envoy/envoy.template.yaml:90]: Why is this being added?

**Review by djwhitt — CHANGES_REQUESTED** [review]

File: envoy/envoy.template.yaml (hunk 2 above)
Comment (hlolli) [reply envoy/envoy.template.yaml]: removed

File: envoy/envoy.template.yaml (hunk 1 above)
Comment (hlolli) [reply envoy/envoy.template.yaml]: removed

**Review by djwhitt — APPROVED** [review]

**Pull request merged**